src/app/App.js

```diff
@@ -0,0 +1,13 @@
+import MainPage from '../components/mainPage/MainPage';
+import Footer from '../components/footer/Footer';
+
+function App() {
+	return (
+		<div>
+			<MainPage />
+			<Footer />
+		</div>
```

Comment:
если тебе не нужен этот div, а скорее всего он не нужен, то можно просто оставить пустые скобки, это не будет создавать доп узел 

```suggestion
		<>
			<MainPage />
			<Footer />
		</>
```